### PR TITLE
Add RLNv2 to TheWakuNetwork

### PR DESCRIPTION
**Pull request by alrevuelta**
* Add RLNv2 to TheWakuNetwork
* Set parameters as listed [here](https://github.com/waku-org/nwaku/blob/master/waku/factory/networks_config.nim#L31)

### CODE REVIEW

File: waku/standards/core/64/network.md [hunk 1]
```diff
@@ -17,7 +17,7 @@ useful for generalized messaging, privacy-preserving, scalable and
 accessible even to resource-restricted devices.
 We'll refer to this opinionated deployment simply as
 _the public Waku Network_, _the Waku Network_ or, if the context is clear, _the network_
-in the rest of this document.
+in the rest of this document. All The Waku Network configuration parameters are listed [here](https://github.com/waku-org/nwaku/blob/master/waku/factory/networks_config.nim#L31).
```
Comment (jm-clius) [inline waku/standards/core/64/network.md:20]:
```suggestion
in the rest of this document.
All The Waku Network configuration parameters are listed [here](https://github.com/waku-org/nwaku/blob/8bfad3ab453f96ac545c7cb0af06d0c0f34d1356/waku/factory/networks_config.nim#L31).
```

Semantic breaks and permalinks for specs.

File: waku/standards/core/64/network.md [hunk 2]
```diff
@@ -141,8 +141,8 @@ See the section on [autosharding](#autosharding) for more.
 
 ## RLN rate-limiting
 
-The [17/WAKU2-RLN-RELAY](../17/rln-relay.md) protocol uses [32/RLN-V1](../../../../vac/32/rln-v1.md) proofs
-to ensure that a pre-agreed rate limit is not exceeded by any publisher.
+The [17/WAKU2-RLN-RELAY](../17/rln-relay.md) protocol uses [32/RLN-V1](../../../../vac/32/rln-v1.md), extended with [RLN-V2](https://github.com/vacp2p/rfc-index/blob/main/vac/raw/rln-v2.md) proofs
```
Comment (jm-clius) [inline waku/standards/core/64/network.md:144]: Shouldn't this now just be relay "...protocol uses RLN-V2" proofs? In other words, is RLN-V1 still relevant to be mentioned?

File: waku/standards/core/64/network.md [hunk 3]
```diff
@@ -141,8 +141,8 @@ See the section on [autosharding](#autosharding) for more.
...
+to ensure that a pre-agreed rate limit is not exceeded by any publisher. The addition of RLNv2 allows to rate-limit with more granularity, `x` amount of messages every `y` seconds.
```
Comment (jm-clius) [inline waku/standards/core/64/network.md:145]:
```suggestion
to ensure that a pre-agreed rate limit is not exceeded by any publisher.
The addition of RLN-V2 allows rate-limiting with more granularity, `x` amount of messages every `y` seconds.
```

As for `RLN-V2` vs `RLNv2` vs `RLNV2` (here and elsewhere) I have no particular preference as long as we use it consistently

File: waku/standards/core/64/network.md [hunk 4]
```diff
@@ -153,32 +153,25 @@ and at times of high network utilization these messages may not be relayed at al
...
+Nodes MUST _reject_ messages not respecting any of these parameters.
+Nodes SHOULD use Network Time Protocol (NTP) to synchronize their own clocks,
+thereby ensuring valid timestamps for proof generation and validation.
+Publishers to the Waku Network SHOULD register an RLN membership.
 
 ### RLN Proofs
 
 Each RLN member MUST generate and attach an RLN proof to every published message
-as described in [17/WAKU2-RLN-RELAY](../17/rln-relay.md/#publishing).
+as described in [17/WAKU2-RLN-RELAY](../17/rln-relay.md/#publishing) and [RLNV2](https://github.com/vacp2p/rfc-index/blob/main/vac/raw/rln-v2.md).
```
Comment (jm-clius) [inline waku/standards/core/64/network.md:174]:
```suggestion
as described in [17/WAKU2-RLN-RELAY](../17/rln-relay.md/#publishing) and [RLN-V2](https://github.com/vacp2p/rfc-index/blob/main/vac/raw/rln-v2.md).
```

File: waku/standards/core/64/network.md [hunk 5]
```diff
@@ -263,16 +256,16 @@ the relay node MUST _reject_ the message.
 
 If a message contains an RLN proof
 and the zero-knowledge proof is invalid
-according to the verification process described in [32/RLN-V1](../../../../vac/32/rln-v1.md),
+according to the verification process described in [32/RLN-V1](../../../../vac/32/rln-v1.md)/[RLN-V2](https://github.com/vacp2p/rfc-index/blob/main/vac/raw/rln-v2.md?plain=1),
```
Comment (jm-clius) [inline waku/standards/core/64/network.md:259]: Same question as before - shouldn't we rather drop all references to RLN-V1?

**Review by jm-clius — APPROVED** [review]
LGTM! Minor comments below.

File: waku/standards/core/64/network.md (hunk 1 above)
Comment (alrevuelta) [reply waku/standards/core/64/network.md]: done: fd45bb7495c9123dd2f00becf30643db5660e6e3

File: waku/standards/core/64/network.md (hunk 2 above)
Comment (alrevuelta) [reply waku/standards/core/64/network.md]: indeed, rln-v1 is not relevant anymore: fd45bb7495c9123dd2f00becf30643db5660e6e3

File: waku/standards/core/64/network.md (hunk 3 above)
Comment (alrevuelta) [reply waku/standards/core/64/network.md]: fix in fd45bb7495c9123dd2f00becf30643db5660e6e3 using RLN-V2

File: waku/standards/core/64/network.md (hunk 4 above)
Comment (alrevuelta) [reply waku/standards/core/64/network.md]: fixed: fd45bb7495c9123dd2f00becf30643db5660e6e3

File: waku/standards/core/64/network.md (hunk 5 above)
Comment (alrevuelta) [reply waku/standards/core/64/network.md]: dropped the reference: fd45bb7495c9123dd2f00becf30643db5660e6e3

**Pull request merged**